lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -57,8 +57,8 @@ def __init__(
         self.axis_aspect_ratio = (
             min(
                 axis_aspect_ratio,
-                calibrated_image.to_position(calibrated_image.data.shape[1])
-                / calibrated_image.to_seconds(calibrated_image.data.shape[0]),
+                calibrated_image.to_seconds(calibrated_image.data.shape[1])
+                / calibrated_image.to_position(calibrated_image.data.shape[0]),
```

Comment:
Might be nice to put in the docstring that the aspect ratio is defined in the units of the kymo (rather than data or screen pixels).